src/workerd/api/node/util.h

```diff
@@ -212,11 +212,11 @@ class UtilModule final: public jsg::Object {
     kj::String functionName;
     kj::String scriptName;
     int lineNumber;
-    int column;
```

Comment:
Did this already go out in a release? If it did, unfortunately we'll need to support both `column` and `columnNumber` as we have no way of knowing if this will break anyone.

Reply:
Should we add a compat_date for this, or just add a new value?